clusterloader2/cmd/clusterloader.go

```diff
@@ -69,7 +69,7 @@ func initFlags() {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 	pflag.StringVar(&clusterLoaderConfig.ReportDir, "report-dir", "", "Path to the directory where the reports should be saved. Default is empty, which cause reports being written to standard output.")
 	pflag.StringArrayVar(&testConfigPaths, "testconfig", []string{}, "Paths to the test config files")
-	pflag.StringVar(&clusterLoaderConfig.TestOverridesPath, "testoverrides", "", "Path to the config overrides file")
+	pflag.StringArrayVar(&clusterLoaderConfig.TestOverridesPath, "testoverrides", []string{}, "Paths to the config overrides file. The latter overrides take precedence over changes in former files.")
```

Comment:
@krzysied - i think it's ok, but can you just confirm that it will work fine?

Reply:
lgtm